### PR TITLE
Update flycut to 1.8

### PR DESCRIPTION
**Pull request by eunikolsky**
- [x] Commit message includes cask’s name (and new version, if applicable).
- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` left no offenses.

### CODE REVIEW

File: Casks/flycut.rb [hunk 1]
```diff
@@ -1,13 +1,13 @@
 cask 'flycut' do
-  version '1.5'
-  sha256 '0a0d3d91194ba8f4e53a9ad004452cb7b0bc5541322f849a203a75720f875908'
+  version '1.8'
+  sha256 'f7fa218a4fee31208476ff400cd1044f3356bf2200a06e32731572498c7ce42a'
 
-  url "https://github.com/downloads/TermiT/Flycut/flycut#{version}.pkg"
+  url "https://github.com/TermiT/Flycut/releases/download/#{version}/Flycut.app.#{version}.zip"
```
Comment (vitorgalvao) [inline Casks/flycut.rb]: Please add an [`appcast`](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/appcast.md).

Comment (fanquake) [inline Casks/flycut.rb]: Should our test suite pick up the fact that components of the cask are
missing?

Comment (vitorgalvao) [inline Casks/flycut.rb]: It does, but only for mandatory components. An `appcast` is optional, and may not even have existed before for this cask.

Comment (eunikolsky) [inline Casks/flycut.rb]: @vitorgalvao, please check the new PR: https://github.com/caskroom/homebrew-cask/pull/21430

**Comment by Fil** [comment]
should be 1.8.1 now
https://github.com/TermiT/Flycut/releases/tag/1.8.1
thanks

**Comment by Fil** [comment]
Update: maybe wait a bit: 1.8.1 is bugged on macOs Sierra
https://github.com/TermiT/Flycut/issues/115

**Pull request merged**